netflix-roulette/src/store/movies.api.ts

```diff
@@ -0,0 +1,26 @@
+import axios from "axios";
+
+const rootUrl = "http://localhost:4000/";
+
+interface FilteredAndSortedMoviesParams {
+  search: string;
+  filter?: string;
+}
+
+interface SortedMoviesParams {
+  sortBy: string;
+}
+
+export const getMovies = () => axios.get(rootUrl + "movies");
+
+export const getFilteredAndSortedMovies = (
```

Comment:
Your data-fetching methods contain repeating parts of axios setup (baseUrl along with some parameters). You could've set up [axios instance](https://axios-http.com/docs/instance) and _decorate_ that with respect to particular method needs.